calibration/Readme.md

```diff
@@ -0,0 +1,135 @@
+The python based calibration software is supported on the DragonBoard 410c Broad Market Kit as well as Windows 10. It is highly recommended to run the calibration software on the Dragonboard platform.
+
+# Dragonboard Install
+The basic setup of the Broad Market Kit is described [here](https://github.com/analogdevicesinc/aditof_sdk).
+
+Clone this SDCARD image to a 16GB SDCARD and put the SD Card slot of the Dragonboard and power on the board.
+
+## Run Sweep Calibtation For A Given Mode
```

Comment:
Run sweep **calibration**

Reply:
This is outdated. Take a look at the other commits.